libusb/os/windows_winusb.c

```diff
@@ -2685,7 +2697,17 @@ static int winusbx_claim_interface(int sub_api, struct libusb_device_handle *dev
 				return LIBUSB_ERROR_ACCESS;
 			}
 		}
+
+		CloseHandle(handle_priv->interface_handle[iface].dev_handle);
 		handle_priv->interface_handle[iface].dev_handle = handle_priv->interface_handle[0].dev_handle;
+		{
+			HANDLE h_proc=NULL;
+			HANDLE h_out = NULL;
+			h_proc = GetCurrentProcess();
```

Comment:
another nit. you just set this to NULL then set it to GetCurrentProcess(); please set this at the declaration.